core/src/main/java/org/opensearch/sql/ast/expression/DataType.java

```diff
@@ -3,17 +3,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-
 package org.opensearch.sql.ast.expression;
 
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 import org.opensearch.sql.data.type.ExprCoreType;
 
-/**
- * The DataType defintion in AST.
- * Question, could we use {@link ExprCoreType} directly in AST?
```

Comment:
Seems like a weird spot for a question... @Yury-Fridlyand, should I remove?

Reply:
Yes
You can also open a GH ticket to fix that.

Reply:
https://github.com/opensearch-project/sql/issues/1912 Opened